gr-analog/lib/squelch_base_ff_impl.cc

```diff
@@ -33,13 +33,31 @@ squelch_base_ff_impl::squelch_base_ff_impl(const char* name, int ramp, bool gate
     d_state = ST_MUTED;
     d_envelope = d_ramp ? 0.0 : 1.0;
```

Comment:
ah, this isn't your fault at all, but! 

If someone sets `ramp` to something > 0 in the constructor, then `d_envelope` will take different during operation. So far, so good.

Now the user later on calls `set_ramp(0)`. See the problem? In that case, we need to set `d_envelope = 1.0f`. So, please amend your `set_ramp` to check that case :)